operators/src/processing/temporal_raster_aggregation/temporal_aggregation_operator.rs

```diff
@@ -257,14 +257,18 @@ where
     #[allow(clippy::too_many_lines)]
     fn create_subquery_adapter_stream_for_single_band<'a>(
         &'a self,
-        query_rect_to_answer: RasterQueryRectangle,
-        band_idx: u32,
+        query: RasterQueryRectangle,
```

Comment:
cant we use that partial query here?

Reply:
do you mean `PartialQueryRectangle`? If so, then no, because it doesn't include the band but the subquery still needs to know it.